pom.xml

```diff
@@ -534,6 +535,12 @@
                 <version>${flink.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.apache.hadoop</groupId>
+                <artifactId>hadoop-aliyun</artifactId>
+                <version>${hadoop-aliyun.version}</version>
+            </dependency>
```

Comment:
Does this dependency have version requirements for hadoop?

Reply:
It has hadoop-common dependency.

Reply:
I mean: Does this dependency have version requirements for hadoop? 
 https://github.com/apache/incubator-seatunnel/pull/2467/files#diff-d9cdd3b0000fbae0ab6febacb4bc6acc28fd71d4fefb238c058ba3c532f7f43bR42

Reply:
It's require Hadoop version 2.9.x+